fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -36,12 +36,23 @@ class ConditionalSelectionRate(ClassificationMoment):
     where error(A = a) = total_error
     """
 
-    def __init__(self, ratio=1.0):
+    def __init__(self, difference_bound=None, ratio_bound_slack=None, ratio_bound=None):
         """Initialize with the ratio value."""
         super(ConditionalSelectionRate, self).__init__()
-        if not (0 < ratio <= 1):
-            raise ValueError(_MESSAGE_RATIO_NOT_IN_RANGE)
-        self.ratio = ratio
+        if (difference_bound is None) and (ratio_bound_slack is None):
+            raise ValueError("One of difference_bound and ratio_bound_slack is required.")
+        if difference_bound is not None and ratio_bound_slack is not None:
+            raise ValueError("Only one of difference_bound and ratio_bound_slack can be used.")
+        if difference_bound is not None:
+            self.eps = difference_bound
+            self.ratio = 1.0
+        if ratio_bound_slack is not None:
```

Comment:
It could be that someone specifies ratio_bound but not the slack. Perhaps we should check on the bound as opposed to the slack. @MiroDudik do you have thoughts on this?

Reply:
Confirmed. We should check the bound. The slack should have default value `0.0` in the signature of the method as mentioned in the other comment.

Reply:
Sounds good. Updated logic to initialize slack with default value 0.0 and check for ratio_bound instead